gradle/validation/owasp-dependency-check/exclusions.xml

```diff
@@ -51,4 +51,25 @@
     <packageUrl regex="true">^pkg:maven/org\.carrot2\.shaded/carrot2\-guava@.*$</packageUrl>
     <cpe>cpe:/a:google:guava</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[Apache Calcite Avatica has separate releases from Apache Calcite]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.calcite\.avatica/.*@.*$</packageUrl>
+    <cpe>cpe:/a:apache:calcite</cpe>
```

Comment:
This ensures that Apache Calcite CVEs don't get attributed to the Apache Calcite Avatica jars.